code/modules/paperwork/pen.dm

```diff
@@ -137,7 +138,7 @@ var/paperwork_library
 	return "<span style=\"[style];color:[P.color]\">[t]</span>"
 
 /datum/writing_style/pen/New()
-	addReplacement(REG_BBTAG("*"), "<li>")
```

Comment:
Yeah I fucking removed this expression, it was eating my right brackets and I don't know why. You want it back go fix it fucking nerds.